src/fetchedTextProcessing.js

```diff
@@ -23,8 +23,10 @@ function sayFirstWordWithScreenReader(word) {
 }
 
 function processFetchedTextParagraph(paragraphOfText) {
+  // fill up wordSpanArray and comparisonWordsArray, while also
```

Comment:
I think some comments are creeping back. I'm totally ambivalent about that

Reply:
forgot to take them out again, will format it as it was

Reply:
done